src/components/project/list/index.tsx

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
+import { Database } from '../../../database.types';
 import { ProjectCard } from '../card';
 
-export const ProjectsList: React.FC<{ projects: { name: string }[] }> = ({
+type Project = Database['public']['Tables']['project']['Row']
```

Comment:
@killerdaisies I think it'd probably be worth defining some type aliases to make using the types easier. What do you think (and where should they go)?

Reply:
Yeah I think that makes sense for sure, like `domain.ts` or `aliases.ts`?